site/js/ChannelsActor.js

```diff
@@ -0,0 +1,106 @@
+import Actor from './Actor.js'
+import { get, post } from './api.js'
+
+export default class ChannelsActor extends Actor {
+  init() {
+    this.on('update channel list', channels => {
+      // Join the first channel if we're not already in one
```

Comment:
Good idea for now. I wonder if there should be a "not in any channel" state? Discord has something like that; e.g., the friend list. But that might not be necessary/useful.